docs/src/modules/components/AppWrapper.js

```diff
@@ -66,7 +66,11 @@ class AppWrapper extends React.Component {
       jssStyles.parentNode.removeChild(jssStyles);
     }
 
-    if ('serviceWorker' in navigator && process.env.NODE_ENV === 'production') {
+    if (
+      'serviceWorker' in navigator &&
+      process.env.NODE_ENV === 'production' &&
+      window.location.host === 'material-ui-next.com'
```

Comment:
https://material-ui-1dab0.firebaseapp.com/ is receiving too much traffic. Let's cut the costs down.

Reply:
This will also block people using this domain name on an old version. Hopefully, they will notice it and stop using it.